### PR TITLE
Run the Hubspot import during the SQL tasks

### PR DESCRIPTION
**Pull request by jon-betts**
For:

 * https://github.com/hypothesis/report/issues/50
 
Requires:

 * https://github.com/hypothesis/report/pull/53

This PR interleaves the creation of the Hubspot tables and the importing of the data so you can do a complete job in one go.

Previously the only way to get this setup was to:

 * Run create from scratch
 * Run the Hubspot import
 * Run the refresh

This was not the behavior from before and means you'd be left in a broken state until you had completed the entire sequence.

This also re-orders the files a bit to be more sensible.

### CODE REVIEW

File: report/sql_tasks/tasks/report/create_from_scratch/00_schema/01_create_schemas.jinja2.sql [hunk 1]
```diff
@@ -1,3 +1,7 @@
+-- This schema largely exists to dump enums/types into so we can import from
+-- the report schemas in LMS and H without trouble
 DROP SCHEMA IF EXISTS report CASCADE;
-
 CREATE SCHEMA report AUTHORIZATION "{{db_user}}";
+
+DROP SCHEMA IF EXISTS hubspot CASCADE;
+CREATE SCHEMA hubspot AUTHORIZATION "{{db_user}}";
```
Comment (jon-betts) [inline report/sql_tasks/tasks/report/create_from_scratch/00_schema/01_create_schemas.jinja2.sql:7]: Should we be dropping the `public` schema here? The `report` schema isn't the main one.

Should we be using `public` at all?

File: report/sql_tasks/tasks/report/create_from_scratch/01_functions/01_date_functions.sql [hunk 2]
```diff
@@ -54,3 +54,6 @@ AS $$ SELECT
     END
 $$
 LANGUAGE SQL;
+
+-- We need to commit these functions, so they are available to Python scripts
+COMMIT;
```
Comment (jon-betts) [inline report/sql_tasks/tasks/report/create_from_scratch/01_functions/01_date_functions.sql:59]: This is very iffy. Why aren't these available to the scripts anyway? Seems like something is wrong, they are all in the same process with the same connection.

I might have just got things in the wrong order?

File: report/sql_tasks/tasks/report/create_from_scratch/00_schema/01_create_schemas.jinja2.sql (hunk 1 above)
Comment (marcospri) [reply report/sql_tasks/tasks/report/create_from_scratch/00_schema/01_create_schemas.jinja2.sql:7]: It's worth a try but I'd leave it as a separate ticket, we might be relying on some implicit permission on it (and that's probably the best reason to delete it).

File: report/sql_tasks/tasks/report/create_from_scratch/01_functions/01_date_functions.sql (hunk 2 above)
Comment (marcospri) [reply report/sql_tasks/tasks/report/create_from_scratch/01_functions/01_date_functions.sql:59]: I tried without this and both create/refresh seemed to work.

Ideally we could remove this if not needed right now.

**Review by marcospri — APPROVED** [review]

**Pull request merged**